poetry.lock

```diff
@@ -92,7 +98,7 @@ test = ["flake8 (3.7.8)", "hypothesis (3.55.3)"]
 [[package]]
 category = "main"
 description = "A backport of the dataclasses module for Python 3.6"
-marker = "python_version >= \"3.6\" and python_version < \"3.7\""
+marker = "python_version >= \"3.6\" and python_version < \"3.7\" or platform_python_implementation == \"CPython\" and python_version < \"3.7\""
```

Comment:
If I read this correctly, we are forcing python 3.7 and not using 3.8. Why is this the case? All tests run and pass in 3.8. We should allow 3.8.

Reply:
Please ignore this file. This file is internally used by poetry. See only `pyproject.toml`.